client/src/store/bubbles.js

```diff
@@ -15,9 +15,9 @@ const initialState = [];
 
 export const fetchBubbles = () => async dispatch => {
     try {
-      console.log('before start')
+      //console.log('before start')
```

Comment:
@codingwithkat didn't completely remove this, since you're still working on these features